CONTRIBUTING.md

```diff
@@ -1,256 +1,269 @@
+For more information, see
+[jupyterhub/pebble-helm-chart](https://github.com/jupyterhub/pebble-helm-chart).
 
+__Install Pebble__
 
+```shell
+helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
+helm repo update
+helm install pebble jupyterhub/pebble --values dev-config.yaml
+```
 
-### 3: Install or upgrade your local Helm chart
+## 4: Build images, update values, install chart
 
-This repository contains various `Dockerfile`s that are used to build docker
-images in use by the Helm chart. To help us build these docker images only when
-needed as well as update the Helm chart's [`values.yaml`](jupyterhub/values.yaml)
-to use the latest available image, we rely on a command line tool called
+This repository contains various `Dockerfile`s that builds to docker images in
+use by the Helm chart. To help us build these docker images only when needed,
+and to update the Helm chart's [values.yaml](jupyterhub/values.yaml) file to use
+the most recent image, we rely on a command line tool called
 [`chartpress`](https://github.com/jupyterhub/chartpress) that is installed as
 part of [dev-requirements.txt](dev-requirements.txt).
 
 Chartpress is configured through [chartpress.yaml](chartpress.yaml), and will
 only rebuild images if their dependent files in their respective directories or
-chartpress.yaml itself has changed.
-
-Now you will be presented with two options as usual, either you a) use the
-automated script to install or upgrade the Helm chart, or b) you do it on your
-own.
+`chartpress.yaml` itself has changed.
 
-#### a) Automated Helm chart install or upgrade
+1. Ensure the latest git tags are available locally, as `chartpress` use them.
 
-1. Install or upgrade your local Helm chart.
-   
    ```shell
-   ./dev upgrade
+   git fetch origin --tags
    ```
 
-#### b) Manual Helm chart install or upgrade
-
 1. Use [`chartpress`](https://github.com/jupyterhub/chartpress) to rebuild
-   modified images if needed but also update the chart's
+   images that needs to be rebuilt and update the chart's
    [values.yaml](jupyterhub/values.yaml) file with the appropriate image tags.
 
    ```shell
+   # run this if you are using k3s, or generally when your Kubernetes cluster
+   # can make direct use of your locally built images
    chartpress
-   ```
 
-   > **NOTE:** If you use a kind cluster and have built new images that will
-   > only available locally, you must also load them into the kind cluster using
-   > the `kind load docker-image <image:tag>` command.
+   # run this if you are using k3d
+   chartpress --image-prefix=registry.local:5000/ --push
+   ```
 
-1. Use `helm` to install or upgrade your Helm chart.
+1. Use `helm` to upgrade (and install) your Helm chart.
 
    ```shell
-   helm upgrade jh-dev ./jupyterhub --install --namespace jh-dev
+   # note that --cleanup-on-fail is a very good practice to avoid
+   # "<resource name> already exist" in future upgrades following
+   # a failed upgrade.
```

Comment:
To me this sort of informative comment would be easier to read as part of the text above, instead of in the code block.